doc/releases/release-notes-3.7.rst

```diff
@@ -804,6 +811,11 @@ Drivers and Sensors
   * Added support for STM32H5 series.
   * Added support to NXP MCXN947
   * Added driver for Analog Devices MAX32 SoC series.
+  * LiteX I2C driver (:file:`drivers/i2c/i2c_litex.c`):
+
+    * added support for bitrate setting from the devicetree
```

Comment:
If you need to do another push, you could capitalize these 3 "added", and add a dot at the end of the 4 sentences that miss it

Reply:
Done :)